api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/validation/IsValidBehandlingsdager.kt

```diff
@@ -16,7 +16,7 @@ fun isValidBehandlingsdager(behandlingsdager: List<LocalDate>): Boolean {
         return false
     }
     val dager = behandlingsdager.sorted()
-    for (i in 1 ..< dager.size) {
```

Comment:
er det meningen at denne committen skal være med i pren?

Reply:
Det var for å bli kvitt lint-feilen på bygget ja, men kan godt fjerne fra denne PR

Reply:
lint har kjørt rødt hele juli pga denne, litt usikker på hvorfor det ikke har blitt fikset tidligere, men koden virker likt som tidligere (så vidt jeg kan se)